docs/HowToGuides/GettingStarted.md

```diff
@@ -257,7 +253,8 @@ Phew, that's a lot to digest! Now let's proceed to the actual build itself!
 
    > **Note**  
    > If you aren't planning to edit the parts of the compiler that are written
-   > in Swift, pass `--bootstrapping=off` to speed up local development.
+   > in Swift, pass `--bootstrapping=hosttools` to speed up local development.
+   > This requires a recent Xcode and/or swift toolchain to be installed.
```

Comment:
Will this work on Linux?

Reply:
[It should](https://github.com/apple/swift/blob/1ebe7ef6a2c740f878ba727ebe559c234e5157c4/cmake/modules/AddSwift.cmake#L532), but is not enabled on the linux CI.

Reply:
Then I suppose the last sentence should be addressing Linux users, because Xcode is on the list of dependencies for macOS anyway.